tests/functional/object/mcg/test_mcg_replication_target_unreachable.py

```
@@ -0,0 +1,135 @@
+        description = alert.get("annotations", {}).get("description", "")
+        logger.info(f"Alert description: {description}")
+        assert (
+            f"from bucket {source_bucket.name} to bucket {target_bucket.name}"
+            in description
+        ), f"Expected bucket names in description, got: {description}"
+
+        # 5. Delete the source bucket
+        try:
+            source_bucket.delete(verify=True)
+        except CommandFailed as e:
+            logger.warning(f"Failed to delete source bucket: {e}")
```

Ruff (0.15.13)

[warning] Logging statement uses f-string

(G004)